manifest-build-tools/post_test.sh

```diff
@@ -0,0 +1,217 @@
+#!/bin/bash
+
+############################################
+# Post-Test scripts for ova, vagrantBox and docker.
+# This script will up/deploy ova, box and docker images,
+# Then check if RackHD services are good running.
+#
+# Usage: 
+# post_test.sh \
+# --type ova, vagrant or docker
+#
+# ova-post-test need some special parameter of net config and target esxi server
+# --adminIP ***.***.***.***
+# --adminGateway ***.***.***.***
+# --adminNetmask 255.255.255.0
+# --adminDNS ***.***.***.***
+# --datastore some-datastore
+# --deployName ova-for-post-test
+# --ovaFile /someDir/some.ova
+# --vcenterHost ***.***.***.***
+# --ntName user
+# --ntPass password
+# --esxiHost ***.***.***.***
+#
+# vagrant-post-test need some special parameter of boxFile and controlNetwork name
+# --boxFile ./someDir/some.box
+# --controlNetwork vmnet*
+#
+# docker-post-test need some special parameter of docker build record file and cloned RackHD repo
+# --RackHDDir ./someDir/RackHD
+# --buildRecord ./record_file
```

Comment:
can you give an example in code, to show what "record_file" looks like ?

otherwise, people have no idea what its format is